File: app/src/main/java/com/leo/fblogin/app/FBLoginApplication.java
package com.leo.fblogin.app;

import android.app.Application;

import com.facebook.FacebookSdk;

/**
 * Created by Leo on 17/03/16.
 */
public class FBLoginApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        FacebookSdk.sdkInitialize(this);
    }
}

File: app/src/main/kotlin/com/leo/fblogin/app/FBLoginApplication.kt
package com.leo.fblogin.app

import android.app.Application

import com.facebook.FacebookSdk

/**
 * Created by Leo on 17/03/16.
 */
class FBLoginApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        FacebookSdk.sdkInitialize(this)
    }
}
